acquia/acquia_cms
# ACMS-1556: Decouple router patch updated.

**Motivation**
Fixes #ACMS-1556

**Proposed changes**
Fix decouple router patch issue.

**Alternatives considered**
NA

**Testing steps**
- Install site with ACMS-1468-1 branch
- Verify are able to apply decouple router patch successfully
- verify site.

**Merge requirements**
